t/irc-commands.t

```diff
@@ -163,7 +163,7 @@ $server->subtest(
 
     $server->server_event_ok('_irc_event_mode')
       ->server_write_ok(
-      ":localhost 367 superman #convos x!*@* superman!~superman@125-12-219-233.rev.home.ne.jp 1577498687\r\n"
+      ":localhost 367 superman #convos x!*\@* superman!~superman\@125-12-219-233.rev.home.ne.jp 1577498687\r\n"
```

Comment:
Should this change be in this PR?

Reply:
Yes, it doesn’t fail in more recent perls

Reply:
ah i see - nvm then :)